apps/worker/tests/test_membership_reconcile.py

```
@@ -0,0 +1,542 @@
+    assert "status='queued'" in sql
+
+
+def test_handler_requeues_on_roster_incomplete():
+    conn = _FakeConn()
+    with patch(
+        "worker.membership_reconcile.fetch_org_roster",
+        side_effect=membership_reconcile.RosterIncomplete("more than _MAX_PAGES pages"),
+    ), patch("worker.httpx.Client"):
+        worker._handle_reconcile_org_membership(conn, 4, _payload(), 0)
+
+    sql, params = conn._cursor.calls[0]
```

Ruff (0.16.2)

[warning] Unpacked variable `params` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)